README.md

```diff
@@ -1,2 +1,41 @@
 # waiter
-Waiter is a web service platform that runs, manages and automatically scales services without requiring human intervention.
\ No newline at end of file
+
+[![Build Status](https://travis-ci.org/twosigma/waiter.svg)](https://travis-ci.org/twosigma/waiter)
+
+Welcome to Two Sigma's waiter project!
+
+Waiter is a web service platform that runs, manages and automatically scales services without requiring human intervention.
+
+[Waiter Design](waiter/docs/waiter-design-docs.md) is a good place to start to learn more.
+
+## Subproject Summary
+
+In this repository, you'll find two subprojects, each with its own documentation.
+
+* [`waiter`](waiter) - This is the actual web service platform, waiter. It comes with a [JSON REST API](waiter/docs/rest-api.md).
+* [`kitchen`](kitchen) - This is the kitchen application, a test app used by the waiter integration tests.
+
+Please visit the `waiter` subproject first to get started.
+
+## Quickstart
+
+The quickest way to get Mesos, Marathon, and waiter running locally is with [docker](https://www.docker.com/) and [minimesos](https://minimesos.org/). 
+
+1. Install `docker`
+1. Install `minimesos`
```

Comment:
The actual numbers in the markdown source don't matter (see https://daringfireball.net/projects/markdown/syntax#list)